src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -208,6 +191,46 @@ private void handleClientListBySite(RoutingContext rc) {
         }
     }
 
+    private void handleClientByKeyId(RoutingContext rc) {
+        try {
+            String keyId;
+
+            try{
```

Comment:
I wonder if there is a more elegant approach to this that avoids having to nest `try-catch`es. What about something similar to the `clientKey` below like:
```
String keyId = rc.queryParam("keyId").stream().findFirst().orElse(null);

if (keyId == null) {
    ResponseUtil.error(rc, 400, "unable to parse keyId ");
    return;
}
```

Reply:
Ah that's much nicer!